package.json

```diff
@@ -213,5 +213,8 @@
         "mocha": "^7.1.2",
         "source-map-support": "^0.5.19",
         "typescript": "^3.8.3"
+    },
+    "dependencies": {
+        "split-grid": "^1.0.9"
```

Comment:
suggest moving this dependency to the /web/transpiler/package.json file

Reply:
Yeah that makes a lot more sense, I guess I was in the wrong directory when I ran "npm install" 🙈

I moved the dependency to the other package.json file now.